fava/templates/overlays/_transaction_form.html

```diff
@@ -13,8 +13,8 @@
                 <input type="text" name="narration" placeholder="Narration" />
             </div>
             <div class="postings">
-            {% for i in range(2) %}
-                <div class="fieldset posting">
+            {% for i in range(3) %}
+                <div class="fieldset posting{% if i == 0 %} template{% endif %}">
```

Comment:
Previous to this change, if you add new posting rows with the "+" the inputs in those rows were not awesomplete-initialized, so there was no auto-complete of them. 

Background:
When cloning the `.posting`-row after pressing "+", the nodes generated by awesomplete were cloned, and the auto-completion did not "carry over" to those nodes, as the `list="..."`-property got removed by awesomplete.